### ISSUE TITLE
text_ce and mel_ce loss

### ISSUE THREAD

**Issue by weidezhang** [152334H/DL-Art-School#57]
it seems when i train the model, the training loss_text_ce and loss_mel_ce_loss as well as validation loss val_loss_text_ce is trending down as expected but the other validation loss val_loss_mel_ce is trending up. 

The training set and validation are quite similar. The data set is small though about 100 wavs each with length at most 8 seconds. 

Any idea what might cause this  ? Below is my experiment with val_text_ce loss and val_mel_ce loss. 

Thanks, 

![Screenshot from 2023-03-22 10-16-02](https://user-images.githubusercontent.com/455617/226951574-cf0d4325-47c3-4e5a-8de9-9308c710cc4b.png)

**Comment by weidezhang** [comment]
![Screenshot from 2023-03-22 10-23-18](https://user-images.githubusercontent.com/455617/226953321-a9e6bc3c-f422-4f86-94e8-21f1b7ca4d65.png)
gpt_gpt and gpt_encoding loss looks normal though

**Comment by 152334H** [comment]
val_loss_mel_ce is probably the most important one. since you say training loss_mel_ce is going down, I would guess that your model is overfitting on the dataset

**Comment by FurkanGozukara** [comment]
what loss value is good? i written a script myself and prepared a very comprehensive tutorial
hopefully i will post soon on https://www.youtube.com/SECourses

so here my plot

![image](https://user-images.githubusercontent.com/19240467/237042273-5c347a58-d55e-4630-851a-790f79c672fe.png)
